DistributionUpdate/PackageUpdate/currentPackageInfoURLList

```diff
@@ -150,7 +150,7 @@ simpcomp    https://simpcomp-team.github.io/simpcomp/PackageInfo.g
 singular    https://gap-packages.github.io/singular/PackageInfo.g
 SLA         https://gap-packages.github.io/sla/PackageInfo.g
 smallgrp    https://gap-packages.github.io/smallgrp/PackageInfo.g
-Smallsemi   http://www-groups.mcs.st-andrews.ac.uk/~jamesm/smallsemi/PackageInfo.g
+Smallsemi   https://gap-packages.github.io/smallsemi/PackageInfo.g
```

Comment:
Thanks @james-d-mitchell - please confirm that http://www-groups.mcs.st-andrews.ac.uk/~jamesm/smallsemi/PackageInfo.g now has the same new version as https://gap-packages.github.io/smallsemi/PackageInfo.g. Otherwise, you need to use `MOVE` command as explained in the comments in this file.

Reply:
I've added MOVE

Reply:
@james-d-mitchell did you force push?

Reply:
Sorry, should be there now.